### DISCUSSION TITLE
How to create multi-tenant apps in nuxt3?

### DISCUSSION THREAD

**Discussion by Nisthar** [nuxt/nuxt#25994] [Questions]
Nextjs team have a really good example for handling multi tenancy using their `rewrite` function. I found this module https://github.com/hieuhani/nuxt-multi-tenancy But it only uses a single folder `[site]` for routing and you check tenant inside the templates. Is there a better way of handling subdomains in nuxt3? 

I want to add all my application logic inside the `app` subdomain. homepage on the main domain.

**Comment by MichaelJCole** [comment]
https://github.com/nuxt/nuxt/issues/31284
